docs/io/fits/usage/cloud.rst

```diff
@@ -0,0 +1,276 @@
+.. currentmodule:: astropy.io.fits
+
+..  _fits_io_cloud:
+
+Obtaining subsets from cloud-hosted FITS files
+**********************************************
+
+Astropy offers support for extracting data from FITS files stored in the cloud.
+Specifically, the `astropy.io.fits.open` function accepts the ``use_fsspec``
+and ``fsspec_kwargs`` parameters, which allow remote files to be accessed in an
+efficient way using the `fsspec <https://filesystem-spec.readthedocs.io>`__
+package.
+
+``fsspec`` is an optional dependency of Astropy which supports reading
+files from a range of remote and distributed storage backends, such as Amazon
+and Google Cloud Storage.  This chapter explains its use.
+
+.. note::
+
+    The examples in this chapter require ``fsspec`` which is an optional
+    dependency of Astropy.  See :ref:`installing-astropy` for details on
+    installing optional dependencies.
+
+
+Subsetting FITS files hosted on an HTTP web server
+==================================================
+
+A common use case for ``fsspec`` is to read subsets of FITS data from a web
+server which supports serving partial files via the
+`Range Requests <https://en.wikipedia.org/wiki/Byte_serving>`__ feature of the
```

Comment:
Why the double underscore instead of single?

Reply:
so it's a one-off sphinx link and not a reference

Reply:
Question: would using a reference be better practice, or does it not matter? I always feel in-line links are easier to add/edit/remove.

Reply:
no, I think this is great as is.